zilencer/management/commands/populate_db.py

```diff
@@ -326,6 +326,14 @@ def handle(self, **options: Any) -> None:
 
             create_users(zulip_realm, names, tos_version=settings.TOS_VERSION)
 
+            aaron = get_user_by_delivery_email("aaron@zulip.com", zulip_realm)
+            aaron.drafts_syncing_enabled = True
+            aaron.save(update_fields=["drafts_syncing_enabled"])
+
+            zoe = get_user_by_delivery_email("ZOE@zulip.com", zulip_realm)
+            zoe.drafts_syncing_enabled = True
+            zoe.save(update_fields=["drafts_syncing_enabled"])
```

Comment:
I think we can skip this logic and have the feature enabled for all default users.